Add `Limits and billing` page for Lightpanda Cloud (#75)
lightpanda-io/docs: src/content/run-on-lightpanda-cloud/getting-started.mdx

Reviewer: This is not related to a change you made on this PR but what do you think about renaming this section something like "Cloud quickstart" or "Lightpanda Cloud Quickstart" to keep the same terminology as the already present [quickstart](https://lightpanda.io/docs/quickstart) section? Or rename the `Quickstart` to `Get started` also works. I just mean to be consistent 😄

Author: You're right that the naming is inconsistent. I'd rather not touch it in this PR though: /quickstart ends with a cloud section of its own, so the two pages need their scopes redrawn, not just a rename. We were thinking about navigation improvement. Not the right thing to settle there

title: Getting started
description: Start using remote browsers easily with Lightpanda cloud.
---
import { Tabs, Callout } from 'nextra/components'

# Getting started

Lightpanda Cloud gives you a remote browser you reach with an API token. You sign up, generate a token, and point your CDP client at a cloud endpoint. There is no browser process to install or keep alive.

## Create an account and token

> Sign up at [console.lightpanda.io/signup](https://console.lightpanda.io/signup). After you sign up, you receive an email with a link to generate your API token. The link is valid for 7 days.

Reviewer 2: ```suggestion
Sign up at [console.lightpanda.io/signup](https://console.lightpanda.io/signup). After you sign up, you receive an email with a link to generate your API token.
```

I wouldn't mention the links validity here.

Author: done

<Callout type="warning">
Your token is shown once, when you generate it. Save it somewhere safe. If you lose it, generate a new one from the console.
</Callout>

## Connect a browser

Your token authenticates every request. Export it, then point your CDP client at the cloud endpoint. Any client that speaks CDP connects the same way.

```sh copy

> export LPD_TOKEN="your token here"

Reviewer 2: I would move the env var export on each puppeteer/chromedp/playwright tabs, b/c we don't use it for CDP URL and it doesn't make sense for this sepcific (but default) tab.

Author: done

```

> 

Reviewer 2: Maybe it worth mentioning we have a tool in the console to help you configure the connection URL: https://console.lightpanda.io/tools/script-generator

Author: I added:

> Prefer not to hand-write the snippet? The [script generator](https://console.lightpanda.io/tools/script-generator) in the console walks you through your client and region, then hands you a ready-to-paste connection snippet with your token already filled in.

https://github.com/lightpanda-io/docs/pull/75/changes/9fe134a0743b70dd695658f6e80c25810d7b96ad#diff-dcd2ae6f439fbc1f7efa07180f203a6b142c47107e0a51b98c9cedc689adcec6R23

Reviewer: This might be a matter of personal preference, but I generally prefer documentation to be as concise as possible. 

Could we maybe merge those 2 paragraphs without loosing any meaning? 

I think these could be removed:
- "Your token authenticates every request."
- "Any client that speaks CDP connects the same way."

and we could summarize the second paragraph in a single sentence like: "To generate a ready-to-use snippet, use the script generator (https://console.lightpanda.io/tools/script-generator)." or something along those lines 😄 WDYT?

Author: Both sentences you flagged are gone, and I took your wording for the script generator. It sits in a callout now, since it is a shortcut around the tabs rather than an introduction to them.
https://github.com/lightpanda-io/docs/blob/331e20839867495abb166efed49aa4c35fa20684/src/content/run-on-lightpanda-cloud/getting-started.mdx#L21

<Tabs items={['CDP URL', 'Puppeteer', 'Playwright', 'Chromedp']}>
  <Tabs.Tab>
The raw endpoint every CDP client connects to:

```text copy
wss://euwest.cloud.lightpanda.io/ws?token=YOUR_TOKEN&browser=lightpanda
```
  </Tabs.Tab>
  <Tabs.Tab>
```js copy
import puppeteer from 'puppeteer-core';

const browser = await puppeteer.connect({
  browserWSEndpoint: `wss://euwest.cloud.lightpanda.io/ws?token=${process.env.LPD_TOKEN}`,
});
```

See the [Puppeteer page](/usage/cdp/puppeteer) for a complete script.
  </Tabs.Tab>
  <Tabs.Tab>
```js copy
import { chromium } from 'playwright-core';

const browser = await chromium.connectOverCDP(
  `wss://euwest.cloud.lightpanda.io/ws?token=${process.env.LPD_TOKEN}`,
);
```

See the [Playwright page](/usage/cdp/playwright) for a complete script.
  </Tabs.Tab>
  <Tabs.Tab>
```go copy
ctx, cancel := chromedp.NewRemoteAllocator(context.Background(),
    "wss://euwest.cloud.lightpanda.io/ws?token="+os.Getenv("LPD_TOKEN"), chromedp.NoModifyURL,
)
defer cancel()
```

See the [Chromedp page](/usage/cdp/chromedp) for a complete script.
  </Tabs.Tab>
</Tabs>

### Endpoints and browser

> Connect to the region nearest you:

Reviewer 2: nearest your script (it could be nearest you or nearest your server...)

Author: ok, done

- `wss://euwest.cloud.lightpanda.io/ws` (west Europe)
- `wss://uswest.cloud.lightpanda.io/ws` (west US)

> The cloud serves Lightpanda by default. Add `browser=chrome` to run Google Chrome instead, or `browser=lightpanda` to force Lightpanda. To route traffic through a proxy, see [Configure a proxy](/guides/configure-a-proxy).

Reviewer 2: I don't think we should mention chrome. Or we must add chrome is offered for debug purpose only.

Author: I rephrased:

> The cloud serves Lightpanda by default. For debugging, append `browser=chrome` to run Google Chrome instead, since Chrome isn't intended for production traffic. Append `browser=lightpanda` to force Lightpanda explicitly. 

https://github.com/lightpanda-io/docs/pull/75/changes/9fe134a0743b70dd695658f6e80c25810d7b96ad#diff-dcd2ae6f439fbc1f7efa07180f203a6b142c47107e0a51b98c9cedc689adcec6R86

title: Getting started
description: Start using remote browsers easily with Lightpanda cloud.
---
import { Tabs, Callout } from 'nextra/components'

# Getting started

> Lightpanda Cloud gives you a remote browser you reach with an API token. You sign up, generate a token, and point your CDP client at a cloud endpoint. There is no browser process to install or keep alive.

Reviewer: It feels a bit verbose IMO. What about a simple caption like "Connect any supported CDP client to a managed Lightpanda browser." ?

Author: Agreed, that opening was announcing the page rather than saying anything. Cut from 48 words to 26, and it now names the three ways in, not just CDP.
https://github.com/lightpanda-io/docs/blob/331e20839867495abb166efed49aa4c35fa20684/src/content/run-on-lightpanda-cloud/getting-started.mdx#L9

## Create an account and token

> Sign up at [console.lightpanda.io/signup](https://console.lightpanda.io/signup). After you sign up, you receive an email with a link to generate your API token.

Reviewer: Repeating "sign up" two times in a row feels slightly repetitive/mechanical 🤔 what about
```suggestion
Sign up at [console.lightpanda.io/signup](https://console.lightpanda.io/signup). We’ll email you a link to generate your API token.
```

Author: Right, that read mechanically. Applied your suggestion as is.
https://github.com/lightpanda-io/docs/blob/331e20839867495abb166efed49aa4c35fa20684/src/content/run-on-lightpanda-cloud/getting-started.mdx#L13

<Callout type="warning">
Your token is shown once, when you generate it. Save it somewhere safe. If you lose it, generate a new one from the console.
</Callout>

> ## Connect a browser

Reviewer: Could we rename to `Connect to Lightpanda Cloud`/`Connect your CDP client` or something similar? It feels a little more natural and makes it clearer that users are connecting their CDP client to a remote browser 👍🏼

Author: Makes sense. Went with Connect your CDP client to Lightpanda Cloud: naming the service reads better than a vague "a remote browser", and both sides of the connection stay visible.
https://github.com/lightpanda-io/docs/blob/331e20839867495abb166efed49aa4c35fa20684/src/content/run-on-lightpanda-cloud/getting-started.mdx#L19
